js_modules/dagster-ui/packages/ui-core/src/workspace/CodeLocationSource.tsx

```diff
@@ -27,8 +27,12 @@ export const CodeLocationSource = ({metadata}: {metadata: Metadata[]}) => {
   const isGitlab = url.hostname.includes('gitlab.com');
 
   if (!isGithub && !isGitlab) {
-    // Unknown URL type. Just render the text.
```

Comment:
@hellendag: what was the reasoning behind this initially? Am I missing some risk that you were thinking of when you originally wrote this?